### PR TITLE
2020-10-23 PyLadies Global Council: Meeting Minutes (for review)

### PR DESCRIPTION
**Pull request by cecivieira**

### CODE REVIEW

**Comment by reshamas** [comment]
@cecivieira 
I propose changing the name of the file to: 2020-10-23-minutes.md so the notes are in numerical order.

(I am not sure how to change it in the review.  Do you know???)

**Review by reshamas — APPROVED** [review]

File: notes/2020/2020_10_23.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+## Project Updates
+1. Project Comms (Slack: #project-communication / Repo: https://github.com/pyladies/project-communications)
+None this time
+
+2. Project Admin & Logistics (Slack: #project-admin-logistics / Repo: https://github.com/pyladies/project-admin-logisitics )
+None this time
+
+3. Project Tech & Infra (Slack: #project-tech-infra / Repo: https://github.com/pyladies/pyladies )
+None this time
+
```
Comment (trallard) [inline notes/2020/2020_10_23.md:17]: There is also the CoC working team though there have not been updates

Comment (trallard) [inline notes/2020/2020_10_23.md:17]: There is also the CoC team with no updates yet

Comment (reshamas) [reply notes/2020/2020_10_23.md:17]: @trallard Do you know how to submit suggestions via GitHub?  If yes, that would be quickest for @cecivieira to accept here.

Comment (reshamas) [reply notes/2020/2020_10_23.md:17]: @trallard Do you know how to submit suggestions via GitHub?  If yes, that would be quickest for @cecivieira to accept here.

Comment (trallard) [inline notes/2020/2020_10_23.md:17]:
```suggestion
4. Project Code of Conduct
None this time
```

**Pull request merged**